build/centos-stream8/include/install-build-dependencies.sh

```diff
@@ -62,6 +62,7 @@ dnf -y install \
   /usr/lib/rpm/redhat/redhat-annobin-plugin-select.sh )
 
 patch /opt/rh/gcc-toolset-12/root/usr/include/c++/12/bits/stl_vector.h < /include/patch.stl_vector.h.diff
+for f in `find /opt/rh/gcc-toolset-12/root/usr/include/c++/12/ -name gthr-default.h`; do patch $f /tmp/patch.gthr-default.h.diff; done
```

Comment:
Likely wrong path, consider `/include/` instead of `/tmp/`

Reply:
Thanks, good catch.